specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/autoDismissAlertsRules.json

```diff
@@ -0,0 +1,355 @@
+          },
+          {
+            "$ref": "#/parameters/TestRule"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/AutoDismissAlertsRule"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "delete": {
+        "x-ms-examples": {
+          "Delete auto dismiss rule data for a subscription": {
+            "$ref": "./examples/AutoDismissAlertsRules/DeleteAutoDismissAlertsRule_example.json"
+          }
+        },
+        "tags": [
+          "AutoDismissAlertsRules"
+        ],
+        "description": "Delete dismiss alert rule for this subscription.",
+        "operationId": "AutoDismissAlertsRules_Delete",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/SubscriptionId"
+          },
+          {
+            "$ref": "#/parameters/AutoDismissAlertsRuleName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AutoDismissAlertsRulesList": {
+      "type": "object",
+      "description": "Auto dismiss rules list for subscription.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AutoDismissAlertsRule"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page."
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "AutoDismissAlertsRule": {
+      "type": "object",
+      "description": "Describes the auto dismiss rule",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AutoDismissAlertsRuleProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "AutoDismissAlertsRuleProperties": {
+      "type": "object",
+      "description": "describes AutoDismissAlertsRule properties",
+      "properties": {
+        "alertType": {
+          "type": "string",
+          "description": "Type of the auto dismissed alert. For all alert types, use '*'"
```

Comment:
Can this be an enum? If modelAsString=true, you can add values to this in a non-breaking fashion. Are the possible values finite?

Reply:
The values are infinite , so it cannot be an enum